src/util/fields.py

```diff
@@ -350,7 +349,7 @@ def adapter(leaf, is_or, negated):
         )
         for alias_id, defaults_s in cr.fetchall():
             try:
-                defaults = dict(safe_eval(defaults_s))  # XXX literal_eval should works.
+                defaults = dict(literal_eval(defaults_s))
```

Comment:
Potential change of behaviour, LGTM just flagging it it new errors start appearing after the merge.